.github/workflows/unit_test_200gb_CI.yml

```diff
@@ -180,15 +180,16 @@ jobs:
         run: |
           if [[ "${GITHUB_EVENT_NAME}" == "pull_request" ]]; then
             export BASE_BRANCH="${{ github.event.pull_request.base.ref }}"
+            cd marqo
+            echo "Running diff-cover against branch $BASE_BRANCH"
+            git fetch origin $BASE_BRANCH:$BASE_BRANCH
+            diff-cover cov.xml --html-report diff_cov.html --markdown-report diff_cov.md \
+              --compare-branch $BASE_BRANCH
           else
-            export BASE_BRANCH=mainline
+            echo "Skipping diff-cover on Push events"
+            echo "Skipped diff-cover on Push events" > marqo/diff_cov.md
+            touch marqo/diff_cov.html
```

Comment:
Disable diff-cov on push events since the base branch is the same and there's nothing to compare. The PR process we use guarantees the coverage of new code once fail-on-threshold is set. This fixes the build failure on mainline branch.

See a successful run here: https://github.com/marqo-ai/marqo/actions/runs/12474131739. This is triggered manually on the branch (push event), and it skipped the diss-cover run.